.github/workflows/containers/github-action-ci/stage1.Dockerfile

```diff
@@ -36,7 +34,29 @@ RUN cmake -B ./build -G Ninja ./llvm \
   -DCMAKE_INSTALL_PREFIX="$LLVM_SYSROOT" \
   -DLLVM_ENABLE_PROJECTS="bolt;clang;lld;clang-tools-extra" \
   -DLLVM_DISTRIBUTION_COMPONENTS="lld;compiler-rt;clang-format;scan-build" \
-  -DCLANG_DEFAULT_LINKER="lld" \
-  -DBOOTSTRAP_CLANG_PGO_TRAINING_DATA_SOURCE_DIR=/llvm-project-llvmorg-$LLVM_VERSION/llvm
+  -DCLANG_DEFAULT_LINKER="lld"
+
+RUN ninja -C ./build stage2-clang-bolt stage2-install-distribution && ninja -C ./build install-distribution && rm -rf ./build
```

Comment:
Maybe add a comment on why the `rm -rf ./build` is here? I believe it was originally to avoid out of disk space errors. Assuming the depot runners have enough disk, it would still be useful as it probably reduces checkpointing time.

Reply:
I'm inclined to drop it.  I think this is a common pattern when doing something on the final image, but since this is just a builder image, I'm not sure if it makes sense to do.  I don't have a strong opinion though.

Reply:
Seems reasonable enough to me.